2-mandatory/1-writers.js

```diff
@@ -81,7 +84,9 @@ Exercise 3:
 */
 
 function logAliveWritersInTheirForties() {
-  // write your code here
+for (let writers in forties) { 
+if {writers.age >=40 && <=49 && writers.alive === true}     
```

Comment:
Just a recommendation as its very important to keep the indentations right, just so that its visually appealing + other people who read your code can understand your code better

```suggestion
    if {writers.age >=40 && <=49 && writers.alive === true}     
```